src/components/widgets-common/top-buttons/top-buttons.directive.coffee

```diff
@@ -16,6 +16,20 @@ module.controller('CommonTopButtonsCtrl', ($scope, $rootScope, $log, ImpacWidget
 
   $scope.hasInfo = ->
     w && w.content? && w.content.info? && w.content.info.length > 0
+
+  $scope.isCsvExportable = ->
```

Comment:
After your PR on the bolt is updated, you should just check if your widget is a bolt widget: all the bolt widget should be "csv_exportable"